cypress/e2e/donor/choose-replacement-attorneys-task.cy.js

```diff
@@ -13,7 +13,7 @@ describe('Choose replacement attorneys task', () => {
 
         cy.contains('label', 'No').click();
         cy.contains('button', 'Continue').click();
-        
+
```

Comment:
I'm thinking about keeping these but skipping them? They are really useful tests for documenting this flow when trying to debug stuff, as there are just too many pages to keep track of, but they are slow.